R/arguments.R

```diff
@@ -316,9 +318,13 @@ min_cols <- function(num_cols, source) {
     p <- ncol(source)
   }
   if (num_cols > p) {
-    msg <- paste0(num_cols, " columns were requested but there were ", p,
-                 " predictors in the data. ", p, " will be used.")
-    rlang::warn(msg)
+    cli::cli_warn(
+      c(
+        "!" = "{num_cols} column{?s} {?was/were} requested but there were
+               {p} predictors in the data.",
+        "i" = "{p} will be used."
```

Comment:
I think it feels weird to say `6 will be used.`

```suggestion
        "i" = "{p} predictors will be used."
```

Reply:
this also doesn't account for when `p = 1`